packages/strapi-generate-new/lib/utils/db-client-dependencies.js

```diff
@@ -12,6 +12,11 @@ const sqlClientModule = {
 module.exports = ({ scope, client }) => {
   switch (client) {
     case 'sqlite':
+      return {
+        'strapi-connector-bookshelf': scope.strapiVersion,
+        knex: '<0.20.0',
+        [sqlClientModule[client]]: '5.0.0',
```

Comment:
isn't it the latest already ?

Reply:
Latest is the issue, I'm trying to force the working version until it's fixed upstream. 

5.0.0 works
5.0.1 (Latest) doesn't

Reply:
Ok I saw that in the issue... I would hope the will  fix sqlite before we release though. Let's wait a bit as we are not going release soon & the users can fix it easily on their side

Reply:
Just FYI if the users aren't aware of the issue this will cause new project generation to fail on windows

Reply:
Also, they don't patch very often:

![Screenshot_20210118-102846_Chrome.jpg](https://user-images.githubusercontent.com/8593673/104946854-0339cf80-5978-11eb-8b15-62bf676cde43.jpg)

Reply:
Ok I'll merge it for the next release but I won't make a release just right now will wait mid week